cdap-docs/developers-manual/source/building-blocks/namespaces.rst

```diff
@@ -28,8 +28,9 @@ namespaces include:
 The first version of namespaces was introduced in :ref:`CDAP v2.8.0 <release-notes>`, and
```

Comment:
Also another use case would be to support multiple sub-organizations within an/your organization